BUILD.bazel

```diff
@@ -27,3 +27,112 @@ java_binary(
     main_class = "com.bazel_diff.Main",
     runtime_deps = ["@bazel_diff//jar"],
 )
+
+alias(
+    name = "envoy_controller",
+    actual = "//feg/gateway/services/envoy_controller:envoy_controller",
+)
+
+alias(
+    name = "connectiond",
+    actual = "//lte/gateway/c/connection_tracker/src:connectiond",
+)
+
+alias(
+    name = "agw_of",
+    actual = "//lte/gateway/c/core:agw_of",
+)
+alias(
```

Comment:
Please run the buildifier to fix the formatting issues.

Run `./bazel/scripts/run_buildifier.sh format`
Check with `./bazel/scripts/run_buildifier.sh check` that all is good.